### PR TITLE
[#2521] Text plugin width resizable. Text plugin uses percentages for si...

### PR DESCRIPTION
**Pull request by ScottDowne**
...zes.

### CODE REVIEW

File: templates/assets/plugins/text/popcorn.text.css [hunk 1]
```diff
@@ -76,13 +78,68 @@
...
 .text-custom {
   display: inline-block;
-  max-width: 500px;
 }
 
 .text-custom > div > span {
-  padding: 0;
+  display: block;
+  font-size: 1em;
+  width: 100%;
```
Comment (mjschranz) [inline templates/assets/plugins/text/popcorn.text.css:89]: I don't know if I like the default being 100% personally. It felt like... "a lot" when switching to custom when I saw the dragging handles.

File: templates/assets/editors/editorhelper.js [hunk 2]
```diff
@@ -60,6 +60,8 @@
           }
         },
         stop: function( event, ui ) {
+          media = mediaContainer.getBoundingClientRect();
```
Comment (mjschranz) [inline templates/assets/editors/editorhelper.js:67]: Hmm. I guess it makes sense that you would want to wait until the dragging stops here.

Comment (ScottDowne) [inline templates/assets/editors/editorhelper.js:67]: Are we good here?

I did this because the media size can change after initial creation of the editor.

By doing this, we assure we always have up to date sizes.

File: css/embed-shell.less [hunk 3]
```diff
@@ -16,7 +16,7 @@ iframe {
 }
 
 
-@media only screen and ( max-width: @medium ) {
+@media only screen and ( max-width: @small ) {
```
Comment (secretrobotron) [inline css/embed-shell.less]: Why did you change this? It should be `medium` to cover the spectrum of sizes. Note that it says `max-width` and not `min-width` like the others.
